libconsensus/pbft/PBFTEngine.cpp

```diff
@@ -469,8 +470,15 @@ bool PBFTEngine::broadcastMsg(unsigned const& packetType, std::string const& key
 void PBFTEngine::broadcastMsg(dev::h512s const& _targetNodes, bytesConstRef _data,
     unsigned const& _packetType, unsigned const& _ttl)
 {
+    std::shared_ptr<dev::h512s> forwardNodes = nullptr;
+    if (m_enableTTLOptimize)
+    {
+        auto sessions = m_service->sessionInfosByProtocolID(m_protocolId);
+        // get the forwardNodes
+        auto forwardNodes = getForwardNodes(sessions);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'forwardNodes' is assigned a value that is never used.](https://app.codacy.com/app/fisco/FISCO-BCOS/pullRequest?prid=4535962)